features/accessibility.feature

```diff
@@ -2,12 +2,6 @@
 
 Feature: About library tests
 
-    Scenario: Home page accessibility check
```

Comment:
Is there a reason the test has been removed?

Reply:
Yes, the linter raised an issue for this test and after speaking with James we decided to remove it. The error was regarding a function/ selenium test being useless so not doing anything. James mentioned how it was to test something or it wasn't really doing anything =)